superduperdb/base/config.py

```diff
@@ -50,6 +50,7 @@ class Cluster(JSONable):
     dask_scheduler: str = 'tcp://localhost:8786'
     local: bool = True
     backfill_batch_size: int = 100
+    cdc: bool = False
```

Comment:
I don't think we need this

Reply:
Discussed in a recent conversation: we should configure `cdc` project wide, and not on a case-by-case basis.
(Should have a concept of configured tables/ collections for SuperDuperDB)